Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -360,7 +360,11 @@ std::optional<WebCore::IPAddress> NetworkSession::firstPartyHostIPAddress(const
     if (firstPartyHost.isEmpty())
         return std::nullopt;
 
-    return m_firstPartyHostIPAddresses.get(firstPartyHost);
+    auto iterator = m_firstPartyHostIPAddresses.find(firstPartyHost);
```

Comment:
Is this a bug fix?

Reply:
In a sense! I added this (currently unused) helper in the previous patch, and then decided that returning `nullopt` in both of these cases would be cleaner than returning either `nullopt`, or the empty value, _or_ a valid `IPAddress`.